modules/get-started/pages/quick-start.adoc

```diff
@@ -200,12 +442,13 @@ Your message is displayed along with its metadata:
   "offset": 0
 }
 ```
-// (step {"action":"runShell", "command": "rpk topic consume chat-room --num 1", "output": "/Pandas are fabulous!/", "timeout": 10000})
+// (step {"action":"runShell", "command": "rpk topic consume chat-room --num 1 -X user=superuser -X pass=secretpassword", "output": "/Pandas are fabulous!/", "timeout": 10000})
 
+== Connect to the cluster externally
 
-To test external connectivity using your local machine:
+It's often more practical to connect to a remote cluster from your local machine with `rpk`. The `docker-compose.yml` file configured the containers to expose ports on your localhost, so you can communicate with the cluster outside the Docker network. To do so, download `rpk` on your local machine and configure it to connect to your cluster on the exposed ports.
 
-. Install rpk on your local machine:
+. Install `rpk` on your local machine:
 +
 NOTE: The `rpk` binary is not supported on Windows.
 +
```

Comment:
@JakeSCahill see this update: https://github.com/redpanda-data/docs/pull/908#issuecomment-2620073223